libraries/Microsoft.Bot.Builder.Azure.CosmosDb/Microsoft.Bot.Builder.Azure.CosmosDb.csproj

```diff
@@ -19,26 +20,16 @@
     <DebugSymbols>true</DebugSymbols>
   </PropertyGroup>
 
-  <PropertyGroup>
-    <!-- The KeyVault package, picked up as a transitive dependency of the Azure Storage libraries
-        doesn't yet support NetStandard20. I confirmed with the Azure Storage team that this warning
-        is fine, and can be supressed.
-
-        It does appear the Azure SDK team is "in-process" of supporting NetStandard20 as seen in this
-        Commit: https://github.com/Azure/azure-sdk-for-net/commit/b0d42d14bfe92a24996826b2487ba592e644f581
-
-        We cannot apply the no-warn supression directly to the package links below as
-        they're not picked up across transitive dependencies. See this GitHub Issue for details:
-        https://github.com/NuGet/Home/issues/5740
-        -->
-    <NoWarn>$(NoWarn);NU1701</NoWarn>
-  </PropertyGroup>
+  <ItemGroup>
+    <Content Remove="C:\Users\erdahlva\.nuget\packages\microsoft.azure.cosmos\3.23.0\contentFiles\any\netstandard2.0\ThirdPartyNotice.txt" />
+  </ItemGroup>
 
   <ItemGroup>
+    <!-- Force System.Text.Encodings.Web to a safe version. -->
+    <PackageReference Include="System.Text.Json" Version="4.7.2" />
+    <PackageReference Include="System.Text.Encodings.Web" Version="4.7.2" />
```

Comment:
We seem to be using Include="System.Text.Encodings.Web" Version="5.0.1" in Integration.AspNet.Core.csproj, do we need this here? if yes, should we use 5.0.1? or 4.7.2 in the other one?